Form-Controls/styles.css

```diff
@@ -0,0 +1,79 @@
+body{
+    background-color: #c7b5aa;
+    font-family: Verdana, Geneva, Tahoma, sans-serif;
+    font-size: 15px;
```

Comment:
Hi @RihannaP ,
Are you using px for the font-size because you want it to be fixed? Have you considered the benefits of using rem over px?